File: katt/magic/include/java/grader.java
import java.io.*;
import java.util.*;

public class grader {

	public static void main(String[] args) throws IOException {
		BufferedReader r = new BufferedReader(new InputStreamReader(System.in));
		PrintWriter w = new PrintWriter(new BufferedOutputStream(System.out));
		String[] nk = r.readLine().split(" ");
		String[] l1 = r.readLine().split(" ");
		String[] r1 = r.readLine().split(" ");
		int N = Integer.parseInt(nk[0]);
		int K = Integer.parseInt(nk[1]);
		int[] L = new int[N];
		int[] R = new int[N];
		for (int i = 0; i < N; i++) {
			L[i] = Integer.parseInt(l1[i]);
			R[i] = Integer.parseInt(r1[i]);
		}

		lib.out = new int[N];
		int res = magic.magic_score(N, K, L, R);
		if (lib.outcount != N) {
			System.out.println("fail, " + lib.outcount + " tricks");
		} else {
			System.out.println(res);
			for (int s : lib.out)
				System.out.print(s + " ");
			System.out.println();
		}
	}

	public static class lib {
		private static int[] out;
		private static int outcount = 0;
		public static int trick(int ans) {
			if (outcount < out.length)
				out[outcount] = ans;
			outcount++;
		}
	}
}

File: katt1/magic/include/python3/grader.py
#!/usr/bin/python3
import grader
import magic
import sys

_tricks = []
def _grader_init():
    N, K = map(int, sys.stdin.readline().split())
    L = list(map(int, sys.stdin.readline().split()))
    R = list(map(int, sys.stdin.readline().split()))
    res = magic.magic_score(N, K, L, R)
    if len(_tricks) != N:
        print("fail, {} tricks".format(len(_tricks)))
    else:
        print(res)
        print(" ".join(map(str, _tricks)))

def trick(ans):
    _tricks.append(ans)

if __name__ != "__main__":
    grader._grader_init()
